build-tools/create-pkg/create-pkg.csproj

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="CreatePkg" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Does this project need to be added to `Xamarin.Android.sln`?

Reply:
Yes, as that'll bring it into alignment with `create-vsix`.

Reply:
If we add this to `Xamarin.Android.sln` perhaps I should also change the default target back to `Build`? I don't think we want this target to run on every solution build.

Reply:
Yes.  This should follow the same/similar conventions as the `create-vsix.csproj` project.

Reply:
I've changed the default target back to `Build`, and overridden `BuildDependsOn` in `create-pkg.targets` so that a build call will do effectively nothing.